tron/serialize/runstate/statemanager.py

```diff
@@ -161,7 +163,13 @@ def restore(self, job_names, skip_validation=False):
                 for job_name, job_state in jobs.items()
             }
             for result in concurrent.futures.as_completed(results):
-                jobs[results[result]]["runs"] = result.result()
+                try:
+                    jobs[results[result]]["runs"] = result.result()
+                except Exception as e:
+                    log.exception(
+                        f"Unable to restore state for {results[result]} - exiting to avoid corrupting data. Exception: {e}"
```

Comment:
```suggestion
                        f"Unable to restore state for {results[result]} - exiting to avoid corrupting data."
```

log.exception will print out the exception following this log message - no need to include it again :)